### ISSUE TITLE
How add replacement_font option to command 

### ISSUE THREAD

**Issue by Sak1s** [mikehaertl/php-pdftk#281]
Hello everyone

can anyone help me how to pass the **replacement_font** parameter to the following example 

$pdf = new Pdf('/path/example.pdf', [
	'command' => '/usr/bin/pdftk',
	'useExec' => true,
]);

$pdf->fillForm($fields)
	->needAppearances()
	->flatten();

$pdf->saveAs(/path/output.pdf)

**Comment by Sak1s** [comment]
i add   the following before `$pdf->saveAs(/path/output.pdf)`  and seems that is working.

$pdf->getCommand()
->addOption('replacement_font', "/path/arial.ttf");

**Comment by mikehaertl** [comment]
@Sak1s I've added another utility method for this. Could you maybe help testing it (see MR above)?

```php
$pdf = new Pdf('/full/path/to/form.pdf');
$result = $pdf->fillForm($data)
    ->replacementFont('/usr/share/fonts/dejavu/DejaVuSans.ttf')
    ->saveAs('filled.pdf');
```

**Comment by Sak1s** [comment]
Hello @mikehaertl 

I test it and works fine . Thank you

**Comment by mikehaertl** [comment]
Great. I've released 0.12.0 containing this change. Thanks for the pointer to this new method.

**Comment by mikehaertl** [comment]
Hmm, seems like nobody noticed that I forgot to merge my MR before releasing 0.12.0. I've now created 0.12.1 which this time *really* includes this new method.
